### PR TITLE
Fix apache ids

### PR DESCRIPTION
**Pull request by junichi11**
I've searched ids one by one in http://people.apache.org/committer-index.html .
- Deleted ids which don't exist.
- Added ids for some committers.

### CODE REVIEW

File: netbeans.apache.org/src/content/community/who.asciidoc [hunk 1]
```diff
@@ -345,14 +338,11 @@ Committer
   -  http://junichi11.com/ http://twitter.com/@junichi_11 
 
 === Ken Fogel
-  - apache id: kfogel
```
Comment (junichi11) [inline netbeans.apache.org/src/content/community/who.asciidoc]: `kfogel` is Karl Fogel's.

File: netbeans.apache.org/src/content/community/who.asciidoc [hunk 2]
```diff
@@ -345,14 +338,11 @@ Committer
   -  http://junichi11.com/ http://twitter.com/@junichi_11 
 
 === Ken Fogel
-  - apache id: kfogel 
   - Dawson College, Montreal, Quebec, Canada 
   - Java, JavaFX, JavaServer Faces, Education 
   -  https://www.omnijava.com https://www.dawsoncollege.qc.ca/computer-science-technology/ http://twitter.com/@omniprof 
 
 === Kenrik Veenstra 
-Committer
```
Comment (junichi11) [inline netbeans.apache.org/src/content/community/who.asciidoc]: No id.

**Comment by junichi11** [comment]
Hi @vieiro ,
Thank you for your confirmation! I already added them ;)
I added "Committer" to Jim's role but we should add "Incubation Mentor" instead of it, right? (Move "Incubation Mentor" from his list. see Bertrand)

**Comment by vieiro** [comment]
Hi @junichi11 
Jim is indeed an incubator mentor. I think it's good to state tthat.
For a whole list of mentors see, for instance, https://incubator.apache.org/projects/netbeans.html

**Comment by junichi11** [comment]
Hi @vieiro 
I'll change it to "Incubation Mentor". Thanks!

**Comment by junichi11** [comment]
Fixed.

**Comment by vieiro** [comment]
Hi @junichi11 
Looks great to me, but maybe we can wait a little bit awaiting for any new announcements?

**Comment by junichi11** [comment]
Hi @vieiro 
Sure. We need not hurry to apply this change at all :) Any time is OK with me.

**Comment by vieiro** [comment]
Hi @junichi11 
@geertjanw wants the committers to be listed as PPMC members as well, could we do that and mark Laszlo  Kishalmi (id: lkishalmi ) as committer and PPMC member as well?
Then we can merge.

**Comment by junichi11** [comment]
Hi @vieiro 
Fixed :) Could you verify it?

**Comment by vieiro** [comment]
Superb @junichi11 ! Thanks!

**Pull request merged**